gamestonk_terminal/fred/fred_api.py

```diff
@@ -0,0 +1,122 @@
+from fredapi import Fred
+import argparse
+import matplotlib.pyplot as plt
+from gamestonk_terminal.helper_funcs import (
+    parse_known_args_and_warn,
+)
+from gamestonk_terminal.config_terminal import API_FRED_KEY
+
+
+def get_GDP(l_args):
+    fred = Fred(api_key=API_FRED_KEY)
+    parser = argparse.ArgumentParser(
+        add_help=False,
+        prog="GDP",
+        description="""
+                GDP
+            """,
+    )
+    parser.add_argument(
+        "-n",
+        dest="n_to_get",
+        type=int,
+        default=-1,
+        required=False,
+        help="Number of GDP Values to Grab",
+    )
+
+    parser.add_argument(
+        "-s",
+        dest="start_date",
+        type=str,
```

Comment:
You can use the valid_date function in helper_funcs, since it checks for the validity of a date. 

I'd also use the same datetime structure the other functions use: YYYY-mm-dd. 

I think that as a default you should just plot/print all data available, rather than setting a default start time.

Reply:
Changed the date time.

I changed he default time to Jan 1 2015.  I think some of the data goes back into the 90s and beyond, so unless someone specifically wants that much data, I think 5 years is plenty.

Reply:
Good point. In that case you can even go as far as 3 years, would be good to have that thought written somewhere :)